Misc/NEWS.d/next/C API/2019-05-22-17-33-52.bpo-37107.8BVPR-.rst

```diff
@@ -0,0 +1,2 @@
+Update PyObject_CallMethodObjArgs and _PyObject_CallMethodIdObjArgs to use
```

Comment:
Please, use markdown syntax here (for example :c:func:`PyObject_CallMethodObjArgs`) and add "Patch by PyObject_CallMethodObjArgs" at the end.